frontend/src/metabase/components/ItemsTable/BaseTableItem.tsx

```diff
@@ -1,209 +0,0 @@
-import moment from "moment-timezone"; // eslint-disable-line no-restricted-imports -- deprecated usage
```

Comment:
IIRC when i looked into this a couple of weeks ago it appears this was left around from a bad merge conflict resolution. it's actually unused.

Reply:
I'm probably the guilty party. Thanks